packages/logging-winston/src/index.js

```diff
@@ -16,174 +16,153 @@
- */
-
-/**
- * @overview
- * ### configuration
+ * If your app is running on Google Cloud Platform, all configuration and
+ * authentication is handled for you. We also auto-detect the appropriate
+ * resource descriptor to report the log entries against.
  *
- * If your app is running on Google Cloud Platform, we handle all configuation
- * & authentication for you. We also auto-detect the appropriate [resource
- * descriptor]{@link todo://add/link} to report the log entries against.
+ * If you are running your application in another environment, such as locally,
+ * on-premise, or on another cloud provider, you will need to provide additional
+ * configuration.
  *
- * If you are running your application in a different environment, for example,
- * locally, on-premise, or on another cloud provider, you may need to provide
- * some configuration.
+ * @param {object} options - [Configuration object](#/docs). Refer to this link
+ *     for authentication information.
+ * @param {string=} options.logName - The name of the log that will receive
+ *     messages written to this transport. Default: `winston_log`
+ * @param {object=} options.levels - Custom logging levels as supported by
+ *     winston. This list is used to translate your log level to the Stackdriver
+ *     Logging level. Each property should have an integer value between 0 (most
+ *     severe) and 7 (least severe). If you are passing a list of levels to your
+ *     winston logger, you should provide the same list here.
+ * @param {object=} options.defaultLevel - The default log level. Winston will
+ *     filter messages with a severity lower than this.
+ * @param {object=} options.resource - The monitored resource that the transport
+ *     corresponds to. On Google Cloud Platform, this is detected automatically,
+ *     but you may optionally specify a specific monitored resource. For more
+ *     information see the
+ *     [official documentation]{@link https://cloud.google.com/logging/docs/api/reference/rest/v2/MonitoredResource}.
  *
  * @example
- * var logger = new winston.Logger({
- *   levels: winston.config.syslog.levels,
- *   transports: new winston.transports.StackdriverLogging({
- *     projectId: 'grape-spaceship-123',
- *     keyFilename: '/path/to/key.json',
- *     levels: winstong.config.syslog.levels,
- *     level: 'warning', // log at 'warning' and above
- *     resource: {
- *       type: 'global'
- *     }
- *   });
+ * var winston = require('winston');
+ * var stackdriverTransport = require('@google-cloud/logging-winston');
+ *
+ * winston.add(stackdriverTransport, {
+ *   projectId: 'grape-spaceship-123',
+ *   keyFilename: '/path/to/keyfile.json',
+ *   level: 'warning', // log at 'warning' and above
+ *   resource: {
+ *     type: 'global'
+ *   }
  * });
- * logger.emerg('antimatter containment field collapse in 10 seconds');
+ *
+ * winston.emerg('antimatter containment field collapse in 10 seconds');
  */
+function LoggingWinston(options) {
```

Comment:
That seemed wrong to me, because as independent modules, we usually try not to overwrite anything. E.g., if some other module writes their own StackdriveLogging implementation with the same name, they could end up overwriting ours from a dependency of a dependency of a dependency that a user doesn't even know they have.

If it is in fact recommended to add on to `winston.transports`, or I'm just misunderstanding the implications, let me know and I'll switch it back.